### ISSUE TITLE
Mobile Vaccines: Permissions and Services Icon

### ISSUE THREAD

**Issue by RowenaN** [openmsupply/mobile#3632]
## Describe the bug

The vaccines permissions and services icon (when all enabled) is the same as the settings icon (the cog icon).  This means that on the mobile home screen there are two identical icons referring to different things.  

### To reproduce

As above.

### Expected behaviour

N/A

### Proposed Solution

I would recommend changing the vaccines permissions and services "enabled" icon.

### Version and device info

Sorry, don't have access currently but from google docs: https://docs.google.com/document/d/1MD17DWP5BJYVh7KuHsdexHUhSMBB0JWg8TeBAR4_gBY/edit#heading=h.ktu9skw3ggj0 

(Let me know if you need further explanation - RN)

### Additional context

The cog icon is also used within the sensors window of mobile vaccines.  The general settings icon for mobile will NOT be visible at the same time.  

The permissions and services icon is will be visible though so have the same issue of two identical icons on one view.  Changing the permissions and services icon would resolve this issue (and the sensors settings icon keeps it consistent with ColdChain).

**Comment by mark-prins** [comment]
yeah, we discussed at the time and decided that the duplication was ok.
we also couldn't agree on a possible alternative. There were some suggestions on https://github.com/openmsupply/mobile/issues/3376 though nothing that really fits the bill. A check/tick/ok sort of icon wouldn't indicate 'settings' to me. Even though the modal shown isn't really settings, as you cannot disable the permissions with it. Once enabled, the screen is informational only.

**Comment by wlthomson** [comment]
For permissions, how about something like this?

![image](https://user-images.githubusercontent.com/43223637/109573468-036ee280-7b54-11eb-9420-a5322029305c.png)
